src/App.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+  <div id="app">
+    <div id="nav">
+      <router-link to="/">Menu</router-link> |
+      <router-link to="/alphabet">Alfabet</router-link> |
+      <router-link to="/numbers">Cyfry</router-link> |
+      <router-link to="/adding">Dodawanie</router-link>
+    </div>
+    <router-view/>
+  </div>
+</template>
+
+<style lang="scss">
```

Comment:
With Aleksy we suggest create file for global scss. You can add folder in `assets` and put there global styles and variables, mixins etc
In  App.vue you can import global styles